EKF/ekf.cpp

```diff
@@ -184,7 +184,6 @@ bool Ekf::initialiseFilter()
 
 		// reset variables that are shared with post alignment GPS checks
 		_gps_pos_deriv_filt(2) = 0.0f;
-		_gps_alt_ref = 0.0f;
```

Comment:
Shouldn't we put it in the reset function above?

Reply:
@bresch I'm not sure, I'd need to look at that. but you need to make sure that you re-run the gps checks after that so that the origin is set again.

Reply:
@bresch Do we support setting new origins nowdays? Maybe I missed that part.

Reply:
No, we don't support new origins, it's just to reset all the variables in the reset function. It also contains `_NED_origin_initialised = false;` so it will have to pass the checks and set it again.
Not sure if there is any corner case that gets created by not resetting it to zero but I have the feeling that it's safer.